test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -65,7 +65,7 @@ public void LoadHostConfig_DefaultsConfig_WhenFileMissing()
         [Fact]
         public void LoadHostConfig_LoadsConfigFile()
         {
-            var path = Path.Combine(Path.GetTempPath(), @"FunctionsTest\host.json");
+            var path = Path.Combine(TestHelpers.FunctionsTestDirectory, "host.json");
```

Comment:
This doesn't fix anything, right? Just simplifying?

Reply:
Yes. Simplifying. Test failure is fixed by creating FunctionsTest directory.